node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -291,6 +289,15 @@ class NodeRuntime(conf: Configuration, host: String, scheduler: Scheduler) {
       _ <- time.sleep(5.seconds).toEffect
     } yield ()
 
+    val casperLoop: Effect[Unit] =
+      for {
+        _ <- casperConstructor.get map {
+              case Some(casper) => casper.fetchDependencies
+              case None         => ().pure[Effect]
+            }
+        _ <- time.sleep(5.seconds).toEffect
```

Comment:
5 seconds seems pretty short to me. Maybe 30 seconds?

Reply:
Ok